modules/ROOT/pages/workday/workday-connector-studio.adoc

```diff
@@ -42,8 +41,7 @@ New objects trigger
 
 . Click the Trigger card and select *Workday Connector*: +
 A list of object types that you can use to trigger Mule applications with the Workday connector is displayed. +
-In the Workday connector, the input source trigger is named *New Objects Trigger*.  +
-//. Specify the Workday https://community.workday.com/sites/default/files/file-hosting/productionapi/operations/index.html[Operation].
+In the Workday connector, the input source trigger is named *New Objects Trigger*.  
 . Select *Click here to set it up* to set the required fields for the *User Name*, *Password*,
```

Comment:
```suggestion
. Select *Click here to set it up* to complete the required fields *User Name*, *Password*,
```
GLOBALLY be sure to use "click" or "select" as indicated in the style reference.